packages/dd-trace/src/util.js

```diff
@@ -17,7 +17,8 @@ function isError (value) {
   if (value && value.constructor) {
     return value.constructor.name === 'JestAssertionError' ||
       value.constructor.name === 'Error' ||
-      value.constructor.name === 'ErrorWithStack'
+      value.constructor.name === 'ErrorWithStack' ||
+      value.constructor.name === 'AssertionError'
```

Comment:
I think this function needs to be fixed at this point with so many specific checks. Maybe it could just consider that something with a `message` and a `stack` is an error regardless of the name?

Reply:
yeah agree 👍 . I'll do that change